build-support/bin/generate_docs.py

```diff
@@ -209,6 +209,7 @@ def run_pants_help_all() -> dict[str, Any]:
         "pants.backend.docker",
         "pants.backend.docker.lint.hadolint",
         "pants.backend.experimental.go",
+        "pants.backend.experimental.helm",
```

Comment:
This means that the targets will show up in the generated docs for 2.11. I think that's fine. Especially thanks to @kaos's past improvement to say what backend everything comes from, which means people will see it's experimental.

Reply:
yeah, was thinking of leaving it out but then, on a second thought, that part of your reference docs was quite helpful for me when trialing the Scala backend...